### ISSUE TITLE
Where are capabilities actually used?

### ISSUE THREAD

**Issue by mudrd8mz** [berrandonea/moodle-block_catalogue#3]
Such as `'block/catalogue:togglehide'` I can't see where it is actually used.

**Comment by mudrd8mz** [comment]
Ah, I see `$permission = has_capability("block/catalogue:toggle$toggler", $coursecontext);`

**Comment by mudrd8mz** [comment]
Still, there is an issue imho. In toggle.php there is only

```
$usereditor = has_capability("block/catalogue:toggle$toggler", $sitecontext);
```

but then you do not use / check against it.

Additionally, I don't understand why some of these capabilities are checked in the system (site) context.

**Comment by berrandonea** [comment]
Fixed in release 1.1

Thank you for your report.
